dev/dll/Microsoft.UI.Xaml.vcxproj

```diff
@@ -215,7 +215,7 @@
       </AdditionalIncludeDirectories>
       <AdditionalOptions>/std:c++17 /bigobj %(AdditionalOptions)</AdditionalOptions>
       <AdditionalOptions>/Wv:18 /Zm1000 %(AdditionalOptions)</AdditionalOptions>
-      <AdditionalOptions>%(AdditionalOptions) /await</AdditionalOptions>
```

Comment:
>await [](start = 47, length = 5)

Why is /await no longer needed?

Reply:
AnimatedVisualPlayer seems to be using co routines...

Reply:
CppWinRT requires it so it's enabled by their props file now, we don't need to do it explicitly. The coroutine code wouldn't compile without this flag so that helps confirm the await flag is still passed. :)